src/main/kotlin/org/rust/ide/console/RsConsoleCommunication.kt

```diff
@@ -18,6 +18,7 @@ class RsConsoleCommunication(private val consoleView: RsConsoleView) {
 
         val codeFragment = consoleView.codeFragment ?: return
         if (codeFragment.text.isEmpty()) return
+        consoleView.handleEvcxrCommand(codeFragment.text.trim())
```

Comment:
IMO it's better to call `handleEvcxrCommand` only for potential REPL commands, if possible. Am I right that all REPL commands start with `:`?